python/gps/algorithm/algorithm_utils.py

```diff
@@ -15,6 +15,7 @@ def __init__(self):
             'cs': None,  # Sample costs of the current iteration.
             'step_mult': 1.0,  # KL step multiplier for the current iteration.
             'eta': 1.0,  # Dual variable used in LQR backward pass.
+            'demo_list': None # List of demonstrations for the current iteration.
```

Comment:
maybe make a note that this is for cost learning in the comment, or that it's unused if not learning a cost.